.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
+
+test.md
```

Comment:
- **[OPTIONAL]** I marked this requirement as optional but please when you ignore the file doesn't charge to the repository it will save locally, the correct path is `./test.md`